package.json

```diff
@@ -116,18 +115,13 @@
     ]
   },
   "lint-staged": {
-    "*.js": [
+    "*.{js,ts}": [
       "eslint --fix"
     ],
-    "*.{ts,md,json}": [
+    "*.{md,json}": [
       "prettier --write"
     ]
   },
-  "husky": {
-    "hooks": {
-      "pre-commit": "lint-staged"
-    }
-  },
```

Comment:
this configuration style was dropped in Husky v5, which is why it stopped working. The hooks are now configured as separate files in `.husky/`